Workbooks/Images/Logos/cyberpion_logo.svg

```diff
@@ -1,9 +1,39 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 4 40 40">
-  <g>
-    <g>
-      <path d="M38.121,28.7a3.565,3.565,0,0,0-5.033-.147A3.627,3.627,0,0,0,32,30.7a7.4,7.4,0,0,1-.576.62,10.823,10.823,0,0,1-7.768,3.24,10.981,10.981,0,0,1-7.809-3.247,11.169,11.169,0,0,1-1.164-1.4l-.449-.635H8.083l1.2,2.214a16.739,16.739,0,0,0,2.75,3.669,16.46,16.46,0,0,0,23.218,0c.139-.141.271-.28.4-.42V34.7l.042.044v-.051l.023-.022a3.63,3.63,0,0,0,2.236-.966A3.547,3.547,0,0,0,38.121,28.7Z" fill="#3b1c34"/>
-      <path d="M14.683,17.157a13.4,13.4,0,0,1,1.155-1.39,11.087,11.087,0,0,1,7.818-3.223,10.925,10.925,0,0,1,7.776,3.215c.1.1.192.212.289.318a3.635,3.635,0,0,0,.994,2.19A3.543,3.543,0,1,0,37.8,13.328a3.631,3.631,0,0,0-2.173-1.06l-.022.021c-.116-.12-.233-.24-.348-.339a16.378,16.378,0,0,0-23.221-.007,15.782,15.782,0,0,0-2.751,3.635l-1.2,2.214h6.151Z" fill="#3b1c34"/>
-    </g>
-    <path d="M18.656,20.475a2.978,2.978,0,0,0-2.125.864H5.15a2.973,2.973,0,0,0-2.123-.864,3.06,3.06,0,1,0,2.107,5.289H16.55a3.078,3.078,0,0,0,2.106.83,3.06,3.06,0,0,0,0-6.119Z" fill="#5dcfca"/>
-  </g>
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
+	 height="300" width="300" viewBox="0 0 792 612" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:none;}
+	.st1{fill:#133775;}
```

Comment:
In  should not have style tag.

Please follow the guideline mention below:
Validator Rules:

proper formatting
size is under 5k
illegal attributes: data-name, style, xmlns:link
Do not use xink:href - use inline instead
Do not use title tags, as they cannot be localized
Each icon needs to be exported with its Object IDs setting set to Unique. If it’s not, Illustrator will generate a generic ID for each layer in the SVG code. This is problematic because if there is a matching ID anywhere else in product code, the icon will be overwritten and not display

You can also refer the guideline [here](https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md)

Thanks,
Aditi Maheshwari

Reply:
I checked the validation rules - 

> illegal attributes: data-name, style, xmlns:link
This states I can't have a style **attribute**. 
Style **tags** should be OK (I also passed validation, which accepts style tags).

Reply:
Regardless, I removed it.